README.md

```diff
@@ -27,10 +27,20 @@ cordova plugin add https://github.com/samzilverberg/cordova-mixpanel-plugin.git
 **window.mixpanel.people:**
 
 - identify(distinctId, onSuccess, onFail)
-- registerPushToken(apnsOrGcmToken, onSuccess, onFail)
+- registerPushId(registrationId, onSuccess, onFail)
```

Comment:
Sure, agreed - ok then I'll change the name to either:

- setPushId
- setPushNotificationId
- setPushNotificationToken

I'll do it in the next couple days. If you have a preference let me know, otherwise I'll just pick the one I like the most ;)